BootloaderCorePkg/Include/Library/SmbiosInitLib.h

```diff
@@ -0,0 +1,66 @@
+/** @file
+
+  Copyright (c) 2017 - 2019, Intel Corporation. All rights reserved.<BR>
+  This program and the accompanying materials
```

Comment:
Should we put this library into BootloaderCommonPkg in case some common module wants to access it?

Reply:
Moved this library into CommonPkg. Thanks.